### PR TITLE
Fix for issue #47

### PR DESCRIPTION
**Pull request by TitasDas**
Replaced unicode_repr() with __repr__().

### CODE REVIEW

**Comment by TitasDas** [comment]
I apologize for the wrong labelling, it should be fix for issue #48 and not #9.

File: Giveme5W1H/extractor/extractors/cause_extractor.py [hunk 1]
```diff
@@ -129,8 +129,7 @@ def _evaluate_tree(self, tree):
                 sibling = sibling.right_sibling()
 
             # NP-VP-NP pattern found .__repr__()
-	    
-            if sibling.label() == 'VP' and "('NP'" in sibling.__repr__():
+	    if sibling.label() == 'VP' and "('NP'" in sibling.__repr__():
```
Comment (fhamborg) [inline Giveme5W1H/extractor/extractors/cause_extractor.py:131]: it appears that the intendation is not correct here. could you please fix this?

Comment (TitasDas) [reply Giveme5W1H/extractor/extractors/cause_extractor.py]: Sure.

**Comment by fhamborg** [comment]
Hi, the indentation is still not correct, see: 
grafik

I don't have access to my IDE atm, otherwise I would have fixed this minor myself. Can you fix it?

**Pull request merged**